content/automate/ManageIQ/System/Policy.class/vmretirerequest_created.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: VmRetireRequest_created
+    inherits: 
+    description: 
+  fields:
+  - rel6:
+      value: "/Infrastructure/${/#miq_request.resource.ci_type}/Retirement/StateMachines/RetirementRequestApproval/Default"
```

Comment:
@d-m-u Since this instance will be used for cloud and infra, should the first part of the value be the substituted value from parse_provider_category?